Mail/Views/Unavailable Mailbox/LockedMailboxView.swift

```diff
@@ -16,44 +16,64 @@
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
+import InfomaniakDI
 import MailCore
 import MailCoreUI
 import MailResources
 import SwiftUI
 
 struct LockedMailboxView: View {
+    @LazyInjectService private var accountManager: AccountManager
+
     @Environment(\.dismiss) private var dismiss
 
+    let mailbox: Mailbox
+
     var body: some View {
         VStack(spacing: UIPadding.regular) {
             MailResourcesAsset.mailboxError.swiftUIImage
                 .resizable()
                 .scaledToFit()
                 .frame(height: 64)
-            Text(MailResourcesStrings.Localizable.lockedMailboxTitle)
+            Text(attributedString())
                 .textStyle(.header2)
                 .multilineTextAlignment(.center)
             Text(MailResourcesStrings.Localizable.lockedMailboxDescription)
                 .textStyle(.bodySecondary)
                 .multilineTextAlignment(.center)
                 .padding(.vertical, value: .medium)
 
-            Button(MailResourcesStrings.Localizable.buttonClose) {
+            Button(MailResourcesStrings.Localizable.externalDialogConfirmButton) {
                 dismiss()
             }
-            .buttonStyle(.ikLink())
+            .buttonStyle(.ikPlain)
             .controlSize(.large)
             .ikButtonFullWidth(true)
         }
         .padding(.horizontal, value: .medium)
         .padding(.top, value: .regular)
         .matomoView(view: ["LockedMailboxView"])
     }
+
+    func attributedString() -> AttributedString {
+        do {
+            var text = try AttributedString(markdown: MailResourcesStrings.Localizable
```

Comment:
I the AttributedString throws, me might juste want to return the string with no bold text?